solutions/cf-1555B.mdx

```diff
@@ -0,0 +1,341 @@
+---
+id: cf-1555B
+source: CF 1555B
+title: B. Two Tables
+author: David Zhang
+---
+
+<Info>
+Oftentimes, problems will try to trick you. For example, since the judge is asking for a double, you would assume you may need to some advanced geometry or some diagonal movements, but a good rule of thumb is to always stay simple. As the most elegant and more importantly the fastest solutions are the shortest.
+</Info>
+
+## Explanation
+
+To begin, you should never move the inital or first rectangle, in a diagonal motion. 
+If the first table were to move in a diagonal motion, the table could achieve the same result just moving in the x or y direction. 
+(The hypotenuse - the diagonal motion - will always be longer then the other sides - the x or y motion)

```

Comment:
hmm? why does it matter, we're only looking for the distance between the ending and beginning of the table right?

also hypotenuse is only for triangles right?
(if you want the reader to visualize it as a triangle, you should probably clarify)